VMBackup/main/workloadPatch/WorkloadUtils/logRestore.py

```diff
@@ -0,0 +1,114 @@
+import sys
+import threading
+import os
+from time import sleep
+import subprocess
+from datetime import datetime
+import re
+try:
+    import ConfigParser as ConfigParsers
+except ImportError:
+    import configparser as ConfigParsers
+
+class Incremental:
+    def __init__(self):
+        self.name = ""
+        self.cred_string = ""
+        self.baseLocation = ""
+        self.parameterFilePath = ""
+        self.oracleParameter = {}
+        self.backupSource = ""
+        self.crontabLocation = ""
+        self.command = ""
+        self.confParser()
+        self.parameterFileParser()
+
+    def parameterFileParser(self):
+        regX = re.compile(r"\*\..+=.+")
+        parameterFile = open(self.parameterFilePath, 'r')
+        contents = parameterFile.read()
+        for match in regX.finditer(contents):
+            keyParameter = match.group().split('=')[0].lstrip('*\.')
+            valueParameter = [name.strip('\'') for name in match.group().split('=')[1].split(',')]
+            self.oracleParameter[keyParameter] = valueParameter
+        #print(self.oracleParameter)
+
+    #----Start Incremental Restore----#
+    def switchControlFiles(self, backupPath):
+        parsedControlFile = self.oracleParameter['control_files']
+        for location in parsedControlFile:
+            os.system('rm -f '+location)
+            os.system('cp -f '+ backupPath + '/control.ctl ' + location)
+            os.system('chmod a+wrx '+location)
+        print("Switched Control Files")
+
+    def switchArchiveLogFiles(self, backupPath):
+        recoveryFileDest = self.oracleParameter['db_recovery_file_dest']
+        dbName = self.oracleParameter['db_name']
+        for location in recoveryFileDest:
+            os.system('rm -R -f '+ location + '/' + dbName[0] +'/archivelog')
+            os.system('cp -R -f ' + backupPath + '/archivelog ' + location + '/' + dbName[0] + '/archivelog')
+            os.system('chmod -R a+wrx '+ location +'/' + dbName[0] + '/archivelog')
+        print("Switched Archive Log Files")
+
+    def restore(self):
```

Comment:
@shridpant - make this a meaningful name

Reply:
Done